WooCommerce/Classes/Authentication/AuthenticationConstants.swift

```diff
@@ -35,4 +35,18 @@ struct AuthenticationConstants {
         "Log in with your WordPress.com account to manage your WooCommerce stores.",
         comment: "Sign in instructions for logging in with a username and password."
     )
+
+    /// Title of "Continue With WordPress.com" button in Login Prologue
+    //
+    static let continueWithWPButtontitle = NSLocalizedString(
+        "Continue With WordPress.com",
+        comment: "Button title. Takes the user to the login by email flow."
```

Comment:
super nit: since the flow also includes SIWA and Google sign in, would it be worth saying "Takes the user to the login with WordPress.com flow." or something along this line?